Cargo.lock

```diff
@@ -5252,7 +5262,7 @@ dependencies = [
  "chrono",
  "lazy_static",
  "matchers",
- "parking_lot 0.9.0",
+ "parking_lot 0.11.0",
```

Comment:
Not sure what caused this. Some rebase magic. Should I revert this change?

Reply:
yea, please revert it